qa/tasks/deepsea.py

```diff
@@ -45,6 +46,16 @@ def dump_file_that_might_not_exist(remote, fpath):
         pass
 
 
+def check_ceph_test_rpm_version(remote):
+    """Checks rpm version for ceph and ceph-test; logs warning if differs"""
+    ceph_test_ver = get_rpm_pkg_version(remote, pkg="ceph-test")
+    ceph_ver = get_rpm_pkg_version(remote, pkg="ceph")
+    if ceph_test_ver != ceph_ver:
```

Comment:
If both ceph and ceph_test are not installed yet, then both ver variable will be None, and no warnings be produced.

Reply:
ceph-test is installed after deepsea and ceph itself gets in after stage.2